sdk-springboot/src/main/java/io/dapr/springboot/DaprBeanPostProcessor.java

```diff
@@ -44,7 +55,7 @@ public Object postProcessAfterInitialization(Object bean, String beanName) throw
    * Subscribe to topics based on {@link Topic} annotations on the given class and any of ancestor classes.
    * @param clazz Controller class where {@link Topic} is expected.
    */
-  private static void subscribeToTopics(Class clazz) {
+  private static void subscribeToTopics(Class clazz) throws IllegalArgumentException {
```

Comment:
It is a runtimeexception, do not add it to the signature.

Reply:
Hmm, I think I tripped some static analysis rule. I'll remove this and see what happens.